runtime/doc/lua.txt

```diff
@@ -2228,10 +2228,6 @@ set({mode}, {lhs}, {rhs}, {opts})                           *vim.keymap.set()*
                             • buffer: (number or boolean) Add a mapping to the
                               given buffer. When "true" or 0, use the current
                               buffer.
-                            • replace_keycodes: (boolean, default true) When
```

Comment:
I think there needs to be a place that mentions the default value of `replace_keycodes` for `vim.keymap.set()`